cmd/gbounty/bootstrap/bootstrap.go

```diff
@@ -511,6 +516,21 @@ func loadProfiles(
 	return actives, passiveReqs, passiveRes
 }
 
+func filterBySeverity[T profile.SeverityGetter](profiles []T, severity string) []T {
```

Comment:
I don't know if this is implementing the correct behavior.

I feel we want to treat "severities" as "levels". So, if I filter by `"High"` severity, I'd expect to see only those issues with `"High" severity, but if I filter by `"Medium"`, I'd expect to see both, those issues with `"Medium"`, but also those with `"High" severity.

Can you confirm @wagiro?
cc/ @iambouali cause you created the original issue.

Anyhow, @shubhamku044 could you please move this function (or all the filtering functions if you cannot apply the `profile.SeverityGetter` strategy pattern) to the `profile` package, and write at least a unit test that proofs the desired behavior?

Reply:
Okay, then the behavior is correct I guess, but @shubhamku044 you still need to check the `SeverityGetter` approach to satisfy what @wagiro said: "_with any step_".

What do you @shubhamku044 think about defining the interface in a slightly different way, like: `HasIssueWithSeverity(s string) bool`, that for `Request` and `Response` checks if the severity is equals, and for `Active` it goes over all steps.